pkg/database/sms_signing_key.go

```diff
@@ -39,7 +39,7 @@ type SMSSigningKey struct {
 
 // GetKID returns the 'kid' field value to use in signing JWTs.
 func (s *SMSSigningKey) GetKID() string {
-	return fmt.Sprintf("r%dmv%d", s.RealmID, s.ID)
+	return fmt.Sprintf("r%dv%ds", s.RealmID, s.ID)
```

Comment:
Should we be more explicit and just say "sms" in here?

Reply:
sure